Sources/PackageModel/UserToolchain.swift

```diff
@@ -489,6 +483,10 @@ public final class UserToolchain: Toolchain {
         return configuration.swiftCompilerFlags
     }
 
+    public var swiftCompilerPathForManifests: AbsolutePath {
```

Comment:
its a bit odd for me to have an accessory named specifically after the call site. why not call this `swiftCompilerPath`?

Reply:
That's what I did originally, but the issue is that we support having two different compilers if `SWIFT_EXEC_MANIFEST` is being used. So in most case, `swiftCompilerPath` (which also exists) and `swiftCompilerPathForManifests` will be identical, but if that environment variable is set, they'll be different. Maybe we should have a comment to that effect?

Reply:
yikes. yes, a comment could help

Reply:
maybe also name this "customManifestCompilerPath"? to make it clearer?

Reply:
I was trying to keep this kinda similar to `swiftCompilerPath` to convey they're similar and so that they both show up in autocomplete next to each other.